src/main/java/com/gruzini/tennistico/controllers/CourtController.java

```diff
@@ -34,6 +38,7 @@ public String showCourtsPage(final Model model) {
     }
     @PostMapping
     public String processCourtSearch(@RequestParam("selectedCity") final String city, final Model model){
+        if (isNull(city)) throw new CourtNotFoundException();
```

Comment:
to samo co w JoinMatchController

Reply:
tutaj się nie zgodzę. chodzi o sprawdzenie requestParama ktory przychodzi. Zobacz że nigdy nie jest po nim nic szukane. Jest on tylko dodawany do modelu. Jeżeli ktoś z łapy do poda/podmieni na htmlu to nie wyleci żaden exception. Mogę przenieść na warstwę serwisową.

Reply:
i zmienię exception na invalid argument albo illegal action

Reply:
to już ktoś by się bardzo musiał starać ten exception wywalić bo jedyny sposób to podstawienie wartości do html'a do buttona.